src/Components/Image/Render.js

```diff
@@ -30,20 +30,35 @@ function handleImageDimension(value) {
   return value;
 }
 
+function placeholderDimension(value, calculatedValue) {
+  let dim = value;
+  if (!value || value === "" || /%$/.test(value)) {
+    dim = calculatedValue && calculatedValue > 0 ? calculatedValue : "125";
+  }
+  return dim.toString().replace(/\D+/g, "");
+}
+
 const ImagePreview = ({
   classes,
   content: {
-    attrs: { src, height, alt, width, title, url },
+    attrs: { src, height, alt, width, title, url, calculatedWidth },
   },
   internals: { isEditor },
   refCallback,
   reCalculate,
 }: $Props) => {
+  const isPlaceholder = !/^(?:http(s)?:\/\/)/.test(src);
```

Comment:
This won't handle relative urls which theoretically could still work in some applications depending on how the engineers set it up. Make that's ok for now though as it's not the case for our use-case